opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -409,23 +410,99 @@ def use_span(
         yield
 
 
+class ProxyTracer(Tracer):
```

Comment:
I think this should be private as this is not something of part official API but an internal implementation detail to solve an issue.

Reply:
Done